controllers/admin/AdminStoresController.php

```diff
@@ -425,7 +426,7 @@ protected function postImage($id)
         * In case of .jpg images, the actual format inside is decided by ImageManager.
         */
         $sfContainer = SymfonyContainer::getInstance();
-        if ($sfContainer->get('prestashop.core.admin.feature_flag.repository')->isEnabled(FeatureFlagSettings::FEATURE_FLAG_MULTIPLE_IMAGE_FORMAT)) {
+        if ($sfContainer->get(FeatureFlagRepository::class)->isEnabled(FeatureFlagSettings::FEATURE_FLAG_MULTIPLE_IMAGE_FORMAT)) {
```

Comment:
You must use `FeatureFlagStateCheckerInterface` instead of the repository.
```suggestion
        if ($sfContainer->get(FeatureFlagStateCheckerInterface::class)->isEnabled(FeatureFlagSettings::FEATURE_FLAG_MULTIPLE_IMAGE_FORMAT)) {
```

Reply:
thanks @boherm :)